static/js/Header.jsx

```diff
@@ -328,7 +328,7 @@ Header.propTypes = {
   registerAvailableFilters:    PropTypes.func,
   setUnreadNotificationsCount: PropTypes.func,
   handleInAppLinkClick:        PropTypes.func,
-  headerMesssage:              PropTypes.string,
+  headerMessage:              PropTypes.string,
```

Comment:
I generally don't spend time worrying about whitespace (my tooling doesn't show it to me), but I can fix it later if requested.